docs/maintainers/releasing.md

```diff
@@ -1,77 +1,72 @@
+1. Trigger the release docker builds for Linux and Windows container images.
+   - Linux: push a branch named `docker` to `powershell/powershell` repository to trigger the build.
+     Delete the `docker` branch once the builds successful at [powershell docker hub](https://hub.docker.com/r/microsoft/powershell/builds/).
+   - Windows: queue a new build from [PowerShellWindowsDocker](https://dscsolutions.visualstudio.com/PowerShellWindowsDocker).
+1. Verify the generated docker container images. (_could be automated_)
+1. [Update the homebrew formula](#homebrew) for the OSX package.
+   This task usually will be taken care of by the community,
+   so we can wait for one day or two and see if the homebrew formula has already been updated,
+   and only do the update if it hasn't.
+
+> Note: Step 2, 3 and 4 can be done in parallel. Step 5 and 6 can be done in parallel.
+
+## Building Packages
+
+> Note: Linux and Windows packages are taken care of by our [release build pipeline](https://dscsolutions.visualstudio.com/PowerShellWindowsDocker/_Build/index?path=%5C&_a=allDefinitions),
+while the OSX package needs to be built separately on a macOS.
+
+The release build should be started based on the `release` branch.
+The release Git tag won't be created until all release preparation tasks are done,
+so the to-be-used release tag should be passed to the release build as an argument.
+
+> When creating the packages, please ensure that the file path does not contain user names.
 That is, clone to `/PowerShell` on Unix, and `C:\PowerShell` for Windows.
 The debug symbols include the absolute path to the sources when built,
 so it should appear `/PowerShell/src/powershell/System.Management.Automation`,
 not `/home/username/src/PowerShell/...`.
 
-[semver]: http://semver.org/
-[tag]: https://git-scm.com/book/en/v2/Git-Basics-Tagging
-[release]: https://help.github.com/articles/creating-releases/
-
-Building Packages
-=================
+### Packaging Overview
 
 The `build.psm1` module contains a `Start-PSPackage` function to build packages.
-It **requires** that `Start-PSBuild -CrossGen -PSModuleRestore` has been run.
+It **requires** that PowerShell Core has been built via `Start-PSBuild`.
 
-Linux / macOS
-------------
+#### Windows
 
-### Overview
+The `Start-PSPackage` function delegates to `New-MSIPackage` which creates a Windows Installer Package of PowerShell.
+The packages *must* be published in release mode,
+so make sure `-Configuration Release` is specified when running `Start-PSBuild`.
 
-The `Start-PSPackage` function delegates to `New-UnixPackage`.
-This function will automatically deduce the correct version from the most recent annotated tag (using `git describe`).
+It uses the Windows Installer XML Toolset (WiX) to generate a MSI package,
+which copies the output of the published PowerShell files to a version-specific folder in Program Files,
+and installs a shortcut in the Start Menu.
+It can be uninstalled through `Programs and Features`.
```

Comment:
I didn't change this content, and not sure if there are any files that cannot be uninstalled.
@joeyaiello are there any files that cannot be uninstalled?